client/python/lib/test/test_make_ndarray.py

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2021 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import numpy as np
+from ovmsclient.tfs_compat.grpc.tensors import make_ndarray
+from tensorflow.core.framework.tensor_shape_pb2 import *
+
+shape = TensorShapeProto(dim = [TensorShapeProto.Dim(size=3)])
+
```

Comment:
Why are you converting and comparing strings in the tests?
Is it possible to create expected ndarray with expected specification (data, shape, dtype) and compare to it?